yarn.lock

```diff
@@ -6697,7 +6697,7 @@ react-inspector@^1.1.0:
     is-dom "^1.0.5"
 
 "react-instantsearch@file:./packages/react-instantsearch/":
-  version "2.0.0-beta.15"
+  version "2.0.0-beta.16"
```

Comment:
Yep I think its just a matter of running yarn again, should be fixed next release.